packages/eclipse/src/styles/globals.css

```
@@ -79,12 +79,12 @@
     --blur-background: 24px;
     --blur-background-high: 40px;
     --font-mona-sans:
-        "Mona Sans", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+        "Mona Sans VF", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
```

Stylelint (17.3.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)